config/Shared/config_default.php

```diff
@@ -334,6 +335,10 @@
 $config[PropelConstants::ZED_DB_REPLICAS] = json_decode(getenv('SPRYKER_DB_REPLICAS') ?: '[]', true);
 $config[PropelConstants::USE_SUDO_TO_MANAGE_DATABASE] = false;
 
+// >>> DATABASE REPLICA CACHE
+$config[PropelReplicationCacheConstants::IS_REPLICATION_ENABLED] = (bool)$config[PropelConstants::ZED_DB_REPLICAS];
```

Comment:
This setting will enable caching when replicas enabled. But I remember that we will integrate this feature by default as always disabled.
@gechetspr could you please advice how we do in this case?